src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenInteractiveParameters.cs

```diff
@@ -15,7 +16,11 @@ internal class AcquireTokenInteractiveParameters : IAcquireTokenParameters
         public Prompt Prompt { get; set; } = Prompt.SelectAccount;
         public CoreUIParent UiParent { get; } = new CoreUIParent();
         public IEnumerable<string> ExtraScopesToConsent { get; set; } = new List<string>();
-        public bool UseEmbeddedWebView { get; set; }
+
+        /// <summary>
+        /// False for system webUi, true for embedded webUi and null for using the default, which is platform specifc
+        /// </summary>
```

Comment:
the only parameter which needs an explanation... should be enough to have this on the Maybe type? Alternatively there could be a method here which would do this with making the field nullable instead. (I'm ok adding a maybe type, even though I have many questions about how needed it is)